app/build.gradle.kts

```diff
@@ -74,7 +79,11 @@ dependencies {
 
     /* Google Mediation Adapters and Dynamic Price */
     api("com.adsbynimbus.android:extension-google:$nimbusVersion")
-    api("com.google.android.gms:play-services-ads:22.1.0")
+    api("com.google.android.gms:play-services-ads:22.2.0")
+
+    /* Admob and Dynamic Price */
```

Comment:
Please rename it to `Dynamic Adapters for AdMob`

Reply:
addressed in https://github.com/timehop/nimbus-android-sample/pull/52